### PR TITLE
Add generic rule for adding comments to fusion prototype usage

### PR DESCRIPTION
**Pull request by dlubitz**
* Add rule for adding comments to used Prototypes.
* Replace dedicated Neos.Neos:PrimaryContent rule with generic rule 
* Add comments for Neos.Fusion:Attributes

### CODE REVIEW

**Comment by kdambekalns** [comment]
Ok, so this PR broke `neos/rector`, at least for me:

```
$ bin/rector --dry-run

 [ERROR] Expected an instance of this class or to this class among its parents
         "Rector\Core\Contract\Rector\RectorInterface". Got:
         "Neos\Rector\Generic\ValueObject\FusionPrototypeNameAddComment" in
         /…/Packages/Libraries/neos/rector/src/../config/set/contentrepository-90.php
         (which is being imported from "/…/rector.php").
```
 If I go back one commit, it works.

File: config/set/contentrepository-90.php [hunk 1]
```diff
@@ -376,8 +375,12 @@
 
     /**
      * Neos.Neos:PrimaryContent
+     * Neos.Fusion:Attributes
      */
-    $rectorConfig->rule(FusionPrimaryContentRector::class);
+    $rectorConfig->ruleWithConfiguration(FusionPrototypeNameAddComment::class, [
```
Comment (dlubitz) [inline config/set/contentrepository-90.php:380]: @kdambekalns Yeah, sorry. AFAIS there is wrong class used:
This needs to be: `FusionPrototypeNameAddCommentRector::class`

**Comment by kitsunet** [comment]
@kdambekalns with the hard pinned rector version of the package?

File: config/set/contentrepository-90.php (hunk 1 above)
Comment (dlubitz) [reply config/set/contentrepository-90.php:380]: It's pretty sh**ty, that the tests are not covering the "real" configuration.

Comment (kdambekalns) [reply config/set/contentrepository-90.php:380]: Yep, creating a PR right now…

**Pull request merged**